### PR TITLE
Addition of a sample "typical post"

### PR DESCRIPTION
**Pull request by kentonh**
So, I thought this section in the docs would be a good place to put the anatomy of a post .md file.

### CODE REVIEW

File: site/_docs/posts.md [hunk 1]
```diff
@@ -111,6 +111,24 @@ Linking to a PDF for readers to download:
...
+Jekyll can handle many different iterations of the idea you might associate with a "post," however a standard blog style post, including an Title, Layout, Publishing Date, and Categories might look like this:
+
+```
+---
+layout: post
+title:  "Welcome to Jekyll!"
+date:   2015-11-17 16:16:01 -0600
+categories: jekyll update
+---
+You’ll find this post in your `_posts` directory. Go ahead and edit it and re-build the site to see your changes. You can rebuild the site in many different ways, but the most common way is to run `jekyll serve`, which launches a web server and auto-regenerates your site when a file is updated.
```
Comment (DirtyF) [inline site/_docs/posts.md]: use rather `bundle exec jekyll serve`

Comment (kentonh) [inline site/_docs/posts.md]: Updated, but this is pulled directly from the sample post in the current _posts directory in the repo.

**Comment by DirtyF** [comment]
Hi @kentonh, could you 🙏  rebase this before merge?  Thanks!

**Comment by DirtyF** [comment]
@jekyllbot: merge +docs

closes #5735

**Comment by zdroid** [comment]
Ugh what was up with those 250 commits? :confused:

**Comment by DirtyF** [comment]
@zdroid refresh ;)

**Comment by zdroid** [comment]
@DirtyF Yeah, I noticed. Imagine merging 250 commits. :laughing:

**Pull request closed without merging**